ddtrace/contrib/elasticsearch/patch.py

```diff
@@ -20,6 +21,8 @@
 from .quantize import quantize
 
 
+log = logging.Logger(__name__)
```

Comment:
We should use the ddtrace logger

```suggestion
log = get_logger(__name__)
```